src/server/scripts/Kalimdor/CavernsOfTime/BattleForMountHyjal/boss_archimonde.cpp

```diff
@@ -186,35 +186,20 @@ struct npc_doomfire_targetting : public ScriptedAI
 
     void Reset() override
     {
-        _chaseTarget = nullptr;
         ScheduleTimedEvent(5s, [&]
         {
-            if (_chaseTarget)
+            // chases someone else every 5 seconds
+            if (Unit* target = SelectTarget(SelectTargetMethod::Random, 0, 0.0f, true, false))
```

Comment:
![image](https://github.com/azerothcore/azerothcore-wotlk/assets/18535853/f6d78a35-e3fa-4d93-b778-877c6af0821e)

Reply:
void DoCastDoomFire(Unit* target)
```
//hack?
doomfireSpirit->setAttackTimer(BASE_ATTACK, 27000);
doomfire->setAttackTimer(BASE_ATTACK, 27000);
```

Reply:
I don't want them to attack. The fire is an immolation like aura on the creatures